scripts.js

```diff
@@ -0,0 +1,11 @@
+function getTriangleArea(a, h) {
+	if ( (a > 0) && (h > 0) ) {
+		return a * h / 2;
+	} else {
```

Comment:
else is unnecessary, return in line 3 will stop function when conditionals are true

write just `return 'Nieprawidłowe dane';` without else